src/plugins/data/public/ui/dataset_selector/configurator.tsx

```diff
@@ -52,25 +53,50 @@ export const Configurator = ({
       defaultMessage: "I don't want to use the time filter",
     }
   );
-  const [language, setLanguage] = useState<string>(() => {
+  const [selectedLanguage, setSelectedLanguage] = useState<string>('');
+  const indexedViewsService = type?.indexedViewsService;
+  const [selectedIndexedView, setSelectedIndexedView] = useState<string | undefined>();
+  const [indexedViews, setIndexedViews] = useState<IndexedView[]>([]);
+  const [loadingIndexedViews, setLoadingIndexedViews] = useState(false);
```

Comment:
Current OSD naming convention is to prefix these with `is`.
```suggestion
  const [isLoadingIndexedViews, setIsLoadingIndexedViews] = useState(false);
```